algebra/src/fields/tweedle/fr.rs

```diff
@@ -0,0 +1,73 @@
+use crate::{
```

Comment:
We need unit tests for this: you can use the same tests of the other curves in the module.

Reply:
Done, check tests.rs file